tests/test_interactions.py

```diff
@@ -206,7 +210,10 @@ def test_ratings_None_but_its_okay(self, df_for_interactions):
                      check_num_negative_samples_is_valid=False)
 
     def test_ratings_None_but_its_explicit_so_not_okay(self, df_for_interactions):
-        with pytest.raises(ValueError):
+        with pytest.raises(ValueError, match=(
+            'Ratings must be provided to ``ExplicitInteractions`` with ``mat`` or ``ratings``'
+            ' - both cannot be ``None``!'
+        )):
```

Comment:
Since `!` has a specific regex meaning, should we just be safe and wrap this in a `re.escape` as well?